### DISCUSSION TITLE
Automation: copy value from static formula to text filed

### DISCUSSION THREAD

**Discussion by APEWAW** [Budibase/budibase#11929] [Help] [answered]
Hello everyone, 

Thank you for giving me the opportunity to ask you for help.

Here is my issue.

Objective (frontend): 
- allow the user to perform a free search in all the text fields of my database

Idea :
- create a formula field (see attach "Search Helper") to concatenate all the text fields columns > it works
- enable search on this formula field > not possible
- create an automation to copy formula field value to a normal text filed column (see attach "Search") > don't know how to do this/if possible

Please consider that I'm very beginner.

Could you help me?
Thank you!

<img width="546" alt="Capture d’écran 2023-09-29 à 11 25 36" src="https://github.com/Budibase/budibase/assets/17066439/93e8326e-5d08-43c6-8f41-388ae2874591">

<img width="390" alt="Capture d’écran 2023-09-29 à 11 25 49" src="https://github.com/Budibase/budibase/assets/17066439/cf3f4184-0d78-4749-8a3e-c4b304fb0944">

<img width="963" alt="Capture d’écran 2023-09-29 à 11 29 08" src="https://github.com/Budibase/budibase/assets/17066439/cc036599-5130-4177-84a3-4c17e99f0f7e">

**Comment by ConorWebb96** [comment, marked as answer]
Hey @APEWAW,

I've made a really simple app you can download and import [here](https://github.com/Budibase/budibase/files/12782208/copy-static-formula-1696245751434.tar.gz) that demonstrates how to do this. 

There are alternative ways which you could do this e.g. trigger the automation whenever the row is clicked, to update that specific row's field. However, I've opted just to demo an automation that runs through all the rows in the table and copies the static formula value across.

I hope this helps.
